cmd/tikv-ctl/src/main.rs

```diff
@@ -776,9 +797,188 @@ fn compact_whole_cluster(
         handles.push(h);
     }
 
-    for h in handles {
-        h.join().unwrap();
-    }
+    handles.into_iter().for_each(|h| h.join().unwrap());
+}
+
+pub const FLASHBACK_TIMEOUT: u64 = 1800; // 1800s
+pub const WAIT_APPLY_FLASHBACK_STATE: u64 = 100; // 100ms
```

Comment:
They are only used inside the file, it's not necessary to make them public, right?

Reply:
done.